server/service/src/sync/translations/invoice.rs

```diff
@@ -181,6 +181,11 @@ pub struct LegacyTransactRow {
     #[serde(rename = "om_original_shipment_id")]
     #[serde(deserialize_with = "empty_str_as_option_string")]
     pub original_shipment_id: Option<String>,
+
+    #[serde(default)]
+    #[serde(rename = "backdated_datetime")]
```

Comment:
I think shis would be om_backdated_datetime in mSupply

Reply:
[2b93947](https://github.com/msupply-foundation/open-msupply/pull/4914/commits/2b9394723aac585574ed65df47b1477c7e62b5c7)